packages/flutter/lib/src/material/icons.dart

```diff
@@ -25,7 +25,7 @@ import 'package:flutter/widgets.dart';
 ///
 ///  * [Icon]
 ///  * [IconButton]
-///  * [design.google.com/icons](https://design.google.com/icons/)
+///  * <https://design.google.com/icons/>
```

Comment:
this (angle brackets, no trailing period, if the url is alone) is my preferred style for urls in these lists